pom.xml

```diff
@@ -1553,7 +1553,7 @@
         <equinox.osgi.version>3.10.2.v20150203-1939</equinox.osgi.version>
         <equinox.osgi.services.version>3.4.0.v20140312-2051</equinox.osgi.services.version>
         <carbon.kernel.version>5.1.0</carbon.kernel.version>
-        <transport.http.version>6.0.156</transport.http.version>
+        <transport.http.version>6.0.158</transport.http.version>
```

Comment:
transport was updated to 159. we dont need this change.

Reply:
Done